### ISSUE TITLE
some questions about loce

### ISSUE THREAD

**Issue by sanmulab** [fcjian/LOCE#2]
Hi author, thank you for your excellent work! I have some questions: 
1. Can MFS be applied to other loss functions, such as EQLv2 and Seesaw Loss, have you done these experiments? 
2. The decoupling training only trains 6 epochs. Will this be too little? Will training more epochs still improve AP?

**Comment by fcjian** [comment]
1. MFS is independent with the loss function, and can be applied to other loss functions theoretically, but we didn't conduct the experiments on EQLv2 and Seesaw Loss.
2. We fine-tune the model for 6 epochs following the previous works such as BALMS, and we didn't fine-tune the model for more epochs.

**Comment by sanmulab** [comment]
Thanks for your reply!

**Comment by sanmulab** [comment]
Hi! I have one last question: how to use MFS alone for training, and how should I modify the config or code? In addition to replacing the EBL loss function, do I need to modify LoceRoIHead?

**Comment by fcjian** [comment]
> Hi! I have one last question: how to use MFS alone for training, and how should I modify the config or code? In addition to replacing the EBL loss function, do I need to modify LoceRoIHead?

For the config, you should modify the loss configure:
- https://github.com/fcjian/LOCE/blob/0a0194d1b42c89ac062bed5e5fdb7a86ce9b70e4/configs/loce/loce_mask_rcnn_r50_fpn_normed_mask_mstrain_2x_lvis_v1.py#L8-L10

For the code, you should modify the loss inputs:
- https://github.com/fcjian/LOCE/blob/0a0194d1b42c89ac062bed5e5fdb7a86ce9b70e4/mmdet/models/roi_heads/bbox_heads/loce_convfc_bbox_head.py#L214-L220

**Comment by sanmulab** [comment]
Sorry, I still don't know which loss parameters need to be modified. If I just use CrossEntropyLoss+MFS for training, then which parameters should I modify in loce_convfc_bbox_head.py:
losses['loss_cls'] = self.loss_cls( 
     cls_score, 
     labels, 
     label_weights, 
     mean_score, 
     avg_factor=avg_factor, 
     reduction_override=reduction_override)

**Comment by fcjian** [comment]
You should remove `mean_score`.

And you can refer to the code:

- https://github.com/fcjian/LOCE/blob/0a0194d1b42c89ac062bed5e5fdb7a86ce9b70e4/mmdet/models/roi_heads/bbox_heads/bbox_head.py#L232-L237

**Comment by sanmulab** [comment]
Thank you!
